src/client/Microsoft.Identity.Client/AuthScheme/PoP/InMemoryCryptoProvider.cs

```diff
@@ -24,7 +24,7 @@ public InMemoryCryptoProvider()
 
         public string CannonicalPublicKeyJwk { get; private set; }
 
-        public string CryptographicAlgorithm { get => "RS256"; }
+        public string CryptographicAlgorithm { get => "PS256"; }
```

Comment:
Can we add tests to verify the "PS256" value of the `CryptographicAlgorithm`?

Reply:
The test got added to the PR as I was adding this comment seems like :)

Reply:
Sorry my bad, added the test but did not push it yet, yes we now have a test